api/controllers/UserController.js

```diff
@@ -71,7 +67,11 @@ module.exports = {
     userAPI
       .findUserByUsername(req.body.username)
       .then(function gotUser(user) {
-        const checkPass = passwordAPI.checkPass(req.body.password, user.encryptedPassword);
+        console.log();
```

Comment:
teehee
```suggestion
```

Reply:
I saw this when running the tests and was like wtf?
![empty console gif](https://user-images.githubusercontent.com/6520704/190034210-47e70ebb-ec2e-4714-8f2a-4fb81cfa3826.gif)